docs/reference.rst

```diff
@@ -74,6 +74,10 @@ The ``BaseConfigGetter`` class
 
         Retrieve a key from available configuration sources, and parse it as a datetime.timedelta object.
 
+    .. py:method:: getpath(key[, default=Path('.')])
```

Comment:
I notice that the default value here is a `Path` file, whereas it's `str` for non-native Python types (timedelta, list).

What is the rule for choosing the type of default values for a given field type?

Reply:
There are apparently currently no rule. Like @bors-ltd pointed out, both `getlist` & `gettimedelta` are already special snowflakes. Both, `default=Path('.')` and `default='.'` would work fine for me.